app/assets/stylesheets/admin/attachment.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the admin/attachment controller here.
+// They will automatically be included in application.css.
```

Comment:
do you need empty file?

I think you should stop using generators

Reply:
I've added changes in the next commit :)